### ISSUE TITLE
A question about INSPIRE

### ISSUE THREAD

**Issue by CyberJJun** [vitaldb/inspire#1]
In vitals, What is the difference between etgas and etsevo when they appear at same time for one patientID?

**Comment by vitaldb** [comment]
The etgas concentration in the INSPIRE dataset is the concentration of
either isoflurane, sevoflurane, or desflurane.

The exact concentration of sevoflurane can be obtained using etsevo.

**Comment by CyberJJun** [comment]
Thanks for reply!

I notice that the records of etgas are more frequent than etsevo for one id.
So can i think etgas = end-tidal  concentration of sevoflurane sampled at realtime and etsevo = concentration of sevoflurane set on the vaporizer when they appear at same time for one id? 

Because normally only one inhaled anesthetic agent is used during one surgery, etgas and etsevo should represent the concentration in different situation i guess.

Anyway, looking forward to your reply!


<img width="508" alt="1" src="https://github.com/user-attachments/assets/bdce344d-909a-4d9a-865b-54c733b48957" />

**Comment by vitaldb** [comment]
Yes it seems that etgas is a measured one (automatically recorded) and
etsevo is a set one (manually recorded).
